index.css

```diff
@@ -0,0 +1,218 @@
+body {
+    background-color:#1b1b2f;
+    width: auto;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+/* navigation */
+
+header {
+    background-color: #1f4068;
+    color: beige;
+    text-align: center;
+    padding-top: 20px;
+}
+nav {
+    width: auto;
+}
+
+nav a {
+    text-decoration: none;
+    color: beige;
+}
+
+nav a:visited{
+    color: beige;
+}
+
+nav a:hover{
+    color: #e43f5a;
+}
+
+nav h2 {
+    width: 25%;
+    float: left;
+    vertical-align: top;
+    margin-top: 0px;
```

Comment:
When a margin property is set to 0, the measure of unit (i.e. px) is redundant and unnecessary.  Won't break your pages, but visual the eye (and brain) more quickly identify "there's nothing to see here" without the "px"; with the "px", the brain has to quickly confirm the value isn't something like a "9".